Project_3/index.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Abril+Fatface|Nunito" rel="stylesheet">
+    <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
+    <title>A Day In The Life: 3.23.19</title>
+</head>
+
+<body>
+
+    <header>
+        <h1 data-aos="zoom-out-left">A Day In The Life</h1>
+        <h2 data-aos="zoom-out-left">3.23.19</h2>
+    </header>
+
+    <div class="timeline-content">
```

Comment:
Could swap this `div` with a `main` element?
```suggestion
    <main class="timeline-content">
```